### ISSUE TITLE
Rerun option only reruns temp file

### ISSUE THREAD

**Issue by mattgarrish** [daisy/ace-gui#112]
Clicking the Rerun option causes Ace to rerun on the temp folder and data it created, not the original file I selected to validate. As a result, no fixes are reflected in the new report. You have to start a new evaluation again.

For example, here's the console output from the first run:

```
Running Ace on C:\epubs\accessible_epub_3.epub
Ace check complete
Loaded report C:\Users\MATTGA~1\AppData\Local\Temp\tmp-14784NiPiG87A8nHF\accessible_epub_3\report.json
```

And here's what rerun triggers:

```
Running Ace on C:\Users\MATTGA~1\AppData\Local\Temp\tmp-14784NiPiG87A8nHF\accessible_epub_3\_unzipped_EPUB_
Ace check complete
Loaded report C:\Users\MATTGA~1\AppData\Local\Temp\tmp-14784NiPiG87A8nHF\accessible_epub_3\report.json```
```

**Comment by danielweck** [comment]
Related issue: https://github.com/daisy/ace-gui/issues/106

**Comment by danielweck** [comment]
I agree that the "re run" feature is confusing, we need to fix this.
That being said, this is a design feature, not a bug.
Ace App is now an editor (accessibility metadata), so the application can be used to iterate over consecutive edits (and associated evaluation re-runs) which necessitates a stable filesystem context folder for the current editing session. The "re-run" command is used for that.
Perhaps the legacy "re-run" command should be reinstated, but I am not sure what the best UX would be for this ... Ace App users must be able to easily differentiate between re-running on a user-controlled fixed-location zipped EPUB (assuming said EPUB container gets recreated by a process external to Ace), versus re-running on the Ace-controlled unzipped EPUB.

In the meantime, re-running the external EPUB can be achieved by drag-drop or via the file system chooser.

Note to devs: the temp directory contains "stable" folder locations for publications with the same filename. This is problematic in cases where a re-run can override previous edits. We should use UUIDs to minimize filename collisions.

**Comment by mattgarrish** [comment]
> Ace App is now an editor (accessibility metadata), so the application can be used to iterate over consecutive edits

Right, I'd forgotten about this until going back over the metadata tab today.

Unless you use that feature, though, it's not intuitive why re-running Ace wouldn't re-run the actual file you're working on.

Given that fixing violations and rerunning reports is more common than editing accessibility metadata and rerunning, it makes the less common need the more easily achieved.

It might make sense to have a localized "rerun" button with the set of metadata editing buttons, at least until there are greater editing facilities available. The application-level re-run would operate on whatever file was used to initiate the checking. A thought, anyway.

**Comment by danielweck** [comment]
i agree with everything you say here. we must eliminate the ambiguity, restore re-run for external zipped epubs, all of which will require some changes in how unzipped publications are stored in temp storage (also see issue #106)
